feat(shouldDropdownOpen): introduce option to open the menu programmatically (#8)
francoischalifour/autocomplete.js: README.md

This turns experience is "controlled" mode. You'll be in charge of updating the state with the [top-level API](#autocomplete).

#### `shouldDropdownOpen`

> > `(options: { state: AutocompleteState }) => boolean`

Reviewer: ```suggestion
> `(options: { state: AutocompleteState }) => boolean` | defaults to `({ state }) => state.results.some(result => result.suggestions.length > 0)`
```

> Needed to check if the dropdown should open depending on the Autocomplete state.

Reviewer: ```suggestion
Called to check whether the dropdown should open based on the Autocomplete state.

The default behavior is to open the dropdown when there are results.
```

Called when an item is selected.

#### `classNames`

> `ClassNames`

> CSS classes applied to the template after the source.

Reviewer: ```suggestion
CSS classes applied to the template of the source.
```
